README.md

```diff
@@ -194,7 +196,7 @@ Implement the function named `highestScoreFrom` in `src/adagrams.js`. This metho
 
 This Wave has 3 parts:
 
-1. Adjust the syntax for the tests for `highestScoreFrom` to run, instead of skip. To do this, find the `describe` block for the tests of `highestScoreFrom`, and change the syntax from `describe.skip` to `describe`
+1. Adjust the syntax for the tests for `highestScoreFrom` to run, instead of skip. To do this, find the `describe` block for the tests of `highestScoreFrom`, and change the syntax from `describe.skip` to [](README.md)`describe`
```

Comment:
```suggestion
1. Adjust the syntax for the tests for `highestScoreFrom` to run, instead of skip. To do this, find the `describe` block for the tests of `highestScoreFrom`, and change the syntax from `describe.skip` to [`describe`](README.md)
```